lib/webauthn/public_key_credential.rb

```diff
@@ -20,7 +20,7 @@ def initialize(type:, id:, raw_id:, client_extension_outputs: {}, response:)
       @type = type
       @id = id
       @raw_id = raw_id
-      @client_extension_outputs = client_extension_outputs
+      @client_extension_outputs = client_extension_outputs || {}
```

Comment:
Why we need this here, this is the default in the argument definition.
Is it for some specific case in which the keyword argument is passed explicitly with `nil`?
If so, why we still want to override with `{}`? Shouldn't the caller be the one who needs to specify `{}` then?

Reply:
> Is it for some specific case in which the keyword argument is passed explicitly with nil?

No, it's just an attempt to normalize the output, related to the talk that we discuss [here](https://github.com/cedarcode/webauthn-ruby/pull/317#discussion_r442290709).
Besides, this helped me to avoid doing something like `client_extension_outputs&.[](:appid)` in [this line](https://github.com/cedarcode/webauthn-ruby/blob/e926b6a1f9a58419c96911be460215eefe60d738/lib/webauthn/public_key_credential_with_assertion.rb#L20). 

But after we agreed on not normalize the output yet, I will go ahead an change it 🙂